packages/odyssey-react/src/components/TextInput/TextInput.module.scss

```diff
@@ -81,16 +80,26 @@
   }
 }
```

Comment:
I believe we can remove this search style now that we are styling this using the pseudo input div and flex